### DISCUSSION TITLE
running `setup.sh`

### DISCUSSION THREAD

**Discussion by georgemilosh** [georgemilosh/Climate-Learning#56] [Q&A] [answered]
It seems running this script may take more than half an hour. Is it possible the order of the packages can be optimised?

**Comment by georgemilosh** [comment]
Ok has been running for more than an hour and half and getting stuck with package conflicts.

**Comment by AlessandroLovo** [comment, marked as answer]
Oh wow! this should not be the case...

**Reply by georgemilosh** [reply]
I already modified this script. The main issue was that tensorflow-gpu with conda is no longer supported, unless you specify the 2.6 verson for example.
